packages/flutter/lib/src/material/page.dart

```diff
@@ -233,6 +248,10 @@ class MaterialPageRoute<T> extends PageRoute<T> {
         );
       case TargetPlatform.iOS:
         return new _CupertinoPageTransition(
+          // Use a linear curve when controlled by a user gesture. This ensures
+          // the animation tracks the user's finger 1:1.
+          // See https://github.com/flutter/flutter/issues/5664
+          curve: userGesture ? null : Curves.fastOutSlowIn,
```

Comment:
on iOS what happens when you fling it? Does it then follow a curve or simulation?

what happens when you let go? Won't this make the page jump?